ДЗ. TypeScript. Анотація типів та примітивні типи.ts

```diff
@@ -59,7 +64,7 @@ class Group {
     this._students.push(student);
   }
 
-  showPerformance() {
+  showPerformance(): Student[] {
     const sortedStudents = this.students.toSorted(

```

Comment:
Don't forget to add type annotations for your variables and constants. It can be very helpful, and it's a good opportunity to practice.